cooltools/expected.py

```diff
@@ -417,101 +519,124 @@ def trans_expected(clr, chromosomes, chunksize=1000000, use_dask=False):
     the actual value of expected for every interchromosomal pair.
 
     """
-
-    def n_total_trans_elements(clr, chromosomes):
-        n = len(chromosomes)
-        x = [clr.extent(chrom)[1] - clr.extent(chrom)[0] for chrom in chromosomes]
-        pairblock_list = []
-        for i in range(n):
-            for j in range(i + 1, n):
-                # appending to the list of tuples
-                pairblock_list.append((chromosomes[i], chromosomes[j], x[i] * x[j]))
-        return pd.DataFrame(pairblock_list, columns=["chrom1", "chrom2", "n_total"])
-
-    def n_bad_trans_elements(clr, chromosomes):
-        n = 0
-        # bad bins are ones with
-        # the weight vector being NaN:
-        x = [
-            np.sum(clr.bins()["weight"].fetch(chrom).isnull().astype(int).values)
-            for chrom in chromosomes
-        ]
-        pairblock_list = []
-        for i in range(len(x)):
-            for j in range(i + 1, len(x)):
-                # appending to the list of tuples
-                pairblock_list.append((chromosomes[i], chromosomes[j], x[i] * x[j]))
-        return pd.DataFrame(pairblock_list, columns=["chrom1", "chrom2", "n_bad"])
+    warnings.warn(
+        "`cooltools.expected.trans_expected()` is deprecated in 0.3.2, will be removed subsequently. "
```

Comment:
`trans_expected` is reimplemented using `blocksum_pairwise` and ready for retirement/removal